src/Microsoft.Framework.Runtime/NuGet/Packages/DefaultPackagePathResolver.cs

```diff
@@ -64,7 +64,7 @@ public virtual string GetPackageDirectory(string packageId, SemanticVersion vers
             string directory = packageId;
             if (_useSideBySidePaths)
             {
-                directory = Path.Combine(directory, version.ToString());
```

Comment:
I'll try to eliminate the usage of `ToString` in scenarios other than debugging.